### PR TITLE
docs: Update contributing guide

### PR DESCRIPTION
**Pull request by tumido**
Small update to make the workflow obvious

### CODE REVIEW

**Review by 4n4nd — APPROVED** [review]
/lgtm

**Comment by 4n4nd** [comment]
looks great :+1: thanks! @tumido

**Comment by anishasthana** [comment]
This PR was merged after only one lgtm?

**Comment by 4n4nd** [comment]
> This PR was merged after only one lgtm?

my bad, I approved it :see_no_evil:

**Comment by tumido** [comment]
I think that's because @4n4nd used the "Approve" button... :smile:

File: contributing.md [hunk 1]
```diff
@@ -21,7 +21,40 @@ Please create an issue in this repo if it directly concerns contents within this
...
+
+   ```sh
+   git checkout master # or main
+   git fetch upstream
+   git rebase upstream/master  # or upstream/main
+   git push origin master
+   ```
+
+5. Create a new branch to work on. Create this branch from the default, up to date branch like `master` or `main`.
+6. Install `pre-commit` git hook.
```
Comment (anishasthana) [inline contributing.md:51]: Could we change it to say "Install `pre-commit` git hook by running `pre-commit install`

Comment (tumido) [reply contributing.md:51]: We certainly can! :+1:

**Pull request merged**